src/jobs/test-nut.yml

```diff
@@ -75,6 +94,8 @@ steps:
       name: Nuts
       command: |
         echo "Using node: $(node --version)"
+        echo "Environment Variables:"
```

Comment:
Dump env vars for posterity. The set env task done by circle only displays those env var made available via contexts and project settings.